src/globals/grid/_grid.scss

```diff
@@ -5,8 +5,34 @@
 @import './experimental/experimental';
 @import './experimental/fallback';
 
+//----------------------------------------------------------------------------------
+// EXPERIMENTAL GRID
+// NOTE: Need to ensure no conflicting functions/mixins/variables, until v10 release
+//----------------------------------------------------------------------------------
+@import '../scss/vendor/@carbon/grid/scss/container';
```

Comment:
One idea I had for elements upstream would be to provide a `@carbon/elements` package that had an entrypoint where it requires packages, and then a `bundle.scss` one that includes all of them in the package so that we wouldn't have to worry about re-writing rules. What do you think?

In other words,

```
packages/elements
  - scss
    - index.scss (uses `@import '@carbon/*`)
    - bundle.scss (references local bundle, essentially copies over files)
    - bundle/
      - `@carbon/layout`
```

Reply:
Sounds good if:

1. Consumer still has choice wrt what specific style to import
2. This specific kind of `@import` is turned into relative path or hoisted to `bundle.scss`